airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/engagements.json

```diff
@@ -48,7 +48,7 @@
     },
     "ownerId": {
       "description": "ID of the owner of the engagement.",
-      "type": ["null", "integer"]
+      "type": ["null", "string"]
```

Comment:
The value in Hubspot [documentation](https://developers.hubspot.com/docs/reference/api/crm/engagements/engagement-details#get-an-engagement) is integer. Without a good reason it won't be accepted the change.